### ISSUE TITLE
groceristar/files.test.js

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/groceristar-fetch#260]
at #258 I saw that we have a few consts `methodlist` that have a lot of similar things. do we really need to have them? @tihaami
After reviewing these files, do you think anything else can be improved too?

**Comment by atherdon** [comment]
I think I got the answer to this question. Because we have different tests.
Btw, I think these functions (from `GS/files.test.js`) can be used at other projects too. What do you think if we'll move them into helper.js? please advice

**Comment by atherdon** [comment]
Also, can you check file GS/getGroceriesWithDepIngKey.test.js(yeah, very bad name)? I think this file also can be upgraded with functions, that can reduce the number of lines

**Comment by tihaami** [comment]
i think it can be improved even further by using the function i made, I had commented that function in chickenKyiv/files.test.js, but i'm not sure about that. Moreover, The functions can be placed at one place and accessed from all the other files.test.js (s) . I thought about that, but seeing your method of creating a methodlist first and then testing each part, i preferred to use yours to maintain consistency throughout the code.
I think all the different tests can be placed in one file and accessed from any file we want from the project.

**Comment by atherdon** [comment]
I like to go step by step, because this plugin used in a few projects, and any major changes can easily break things... And as you just started with me - i think it'sa good move slowly introduce you to our workflow and code base

**Comment by tihaami** [comment]
Alright, so right now the next step would be using writing functions in GS/getGroceriesWithDepIngKey.test.js for similar lines of code, and placing similar tests methods in helper.js ?

**Comment by atherdon** [comment]
Yes, please start with that test file, so we'll have few cases for our
functions. Then move similar functions to helper and then update *.test.js
files

>

**Comment by atherdon** [comment]
just to let you know, i'm updating a module, that giving us data files, that we're using. So some part of module can be broken or tests will fail.

**Comment by tihaami** [comment]
Okay, got it. I'll complete the job by tomorrow..

**Comment by atherdon** [comment]
Sure, take your time - we don't have any rush here.
Please don't forget to sync your fork, because I made some changes, and it
breaking some old code.
When you sync it - you'll see what I mean. It actually improves some
readability and simplifies part of the project

>

**Comment by atherdon** [comment]
we made some changes in our package in last days, please sync your fork

**Comment by tihaami** [comment]
Roger..

>
